frontends/bmd/cypress.config.ts

```diff
@@ -3,6 +3,7 @@ import { defineConfig } from 'cypress'
 export default defineConfig({
   viewportHeight: 1080,
   viewportWidth: 1920,
+  defaultCommandTimeout: 8_000,
```

Comment:
TIL about `_` as a number separator! I have a slight preference for reserving these separators for cases where there are greater than 3 zeros (e.g. 10_000) but using them more consistently is fine, too!

Reply:
Ah yeah it does seem most useful with many zeros. I just got into the habit of using them all the time now. It grew on me :D